roles/clamav/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+clamav_image: "clamav/clamav"  
+clamav_container_name: "clamav-scan" 
+clamav_scan_directory: ""
+clamav_log_path: "/var/log/clamav"
+clamav_db_path: "/var/lib/clamav" 
+clamav_enable_auto_scan: false
+clamav_scan_exclude_directories:
+  - "/dev/*"
+  - "/proc/*"
+  - "/sys/*"
+  - "/run/*"
+  - "/tmp/*"
+  - "/mnt/*"
+  - "/media/*"
+  - "/lost+found/*"
+  - "/var/lib/docker/*"  
+  - "/var/lib/containerd/*"  
+  - "/var/lib/nginx/*"  
+  - "/var/lib/systemd/*" 
+  - "/var/lib/snapd/*"  
+  - "/var/lib/fail2ban/*"  
+  - "/var/lib/private/*"  
+  - "/var/lib/ubuntu-release-upgrader/*"  
+  - "/var/lib/tpm/*"  
+  - "/var/lib/update-manager/*"  
+  - "/var/lib/usb_modeswitch/*" 
+clamav_user_id: "2330"
+clamav_group_id: "2330"
+clamav_username: "clamav" 
+clamav_port: "3310" 
+clamav_cron_hour: "22"
+clamav_memory_limit: "2g"        
+clamav_cpu_limit: "0.5"
+clamav_remove: false
+clamav_quarantine: true
+clamav_quarantine_path: "/var/lib/clamav-quarantine"  
+clamav_create_log_directroy: true    
```

Comment:
We should either remove this option or use it in the "Create ClamAV log directory" task (would allow setting custom/different permissions).